### PR TITLE
Added util.csvparsing package for CSV Upload Parsing

### PR DESCRIPTION
**Pull request by njd9419**
- Added CsvFileReader.cs, class for reading CSV files
- Added CsvMonumentConverter.cs, class for converting CSV rows representing Monuments into Monument objects
- Added InitialDataLoadApplication.cs, makes use of util.csvparsing to load the initial dataset into the database

### CODE REVIEW

File: src/main/java/com/monumental/models/Monument.java [hunk 1]
```diff
@@ -104,19 +104,19 @@ public void setMaterial(String material) {
         this.material = material;
     }
 
-    public double getlat() {
+    public double getLat() {
```
Comment (benvogler) [inline src/main/java/com/monumental/models/Monument.java]: I got lazy with the find and replace 😄

File: src/main/java/util/InitialDataLoadApplication.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.List;
+
+/**
+ * Class used to load the initial dataset into the database
+ * Creates a CsvFileReader class and uses it to iterate all of the rows in the CSV
+ * Turns each row into a Monument object, accumulates them and inserts them into the database
+ */
+public class InitialDataLoadApplication {
+    public static void main(String[] args) {
+        String pathToDatasetCsv = "C:\\Users\\nickb\\Documents\\Initial Dataset.csv";
```
Comment (benvogler) [inline src/main/java/util/InitialDataLoadApplication.java]: We should make this configurable somehow

File: src/main/java/util/csvparsing/CsvMonumentConverter.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+public class CsvMonumentConverter {
+    /**
+     * Static method for converting a CSV row representing a Monument into a Monument object
+     * @param csvRow - CSV row representing a Monument, as a String
+     * @return Monument - the Monument object represented by csvRow
+     */
+    public static Monument convertCsvRowToMonument(String csvRow) {
+        // Regex to split on commas only if the comma has zero or an even number of quotes ahead of it
+        // See: https://stackoverflow.com/questions/1757065/java-splitting-a-comma-separated-string-but-ignoring-commas-in-quotes
+        String[] csvRowArray = csvRow.split(",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)", -1);
```
Comment (benvogler) [inline src/main/java/util/csvparsing/CsvMonumentConverter.java]: :oof:

File: src/main/java/util/csvparsing/CsvMonumentConverter.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        Monument result = new Monument();
+
+        for (int columnIndex = 0; columnIndex < csvRowArray.length; columnIndex++) {
+            String value = csvRowArray[columnIndex];
+
+            // NOTE: The order of the columns is specific to the initial dataset
```
Comment (benvogler) [inline src/main/java/util/csvparsing/CsvMonumentConverter.java]: Normally with csv you read in the first line as headers

**Comment by njd9419** [comment]
Yeah, I did the minimum I needed to do to get the data into the system. This will definitely need to be revisited once we do the CSV Upload Parsing feature.

File: src/main/java/util/csvparsing/CsvMonumentConverter.java (hunk 4 above)
Comment (njd9419) [reply src/main/java/util/csvparsing/CsvMonumentConverter.java]: Got lazy :)

**Pull request merged**